### ISSUE TITLE
Windows Support not working

### ISSUE THREAD

**Issue by StephaneMartignier** [vkarpov15/run-rs#13]
Hi,

So trying to use multi-document in MongoDB I found that I need replicat-set and then I find your npm pacakge and I was like "that's perferct :)"

I run the command to install globally and it's seem good. Then I try to run "run-rs -v 4.0.0" but the command failed with the following error :
![run-rs1](https://user-images.githubusercontent.com/15869403/48557532-5373fd80-e8e7-11e8-9b58-f7408f75f6cf.PNG)

My guess will be that there is an error in the command to create the folder for the database?

Or am I doing something wrong?

I have version 0.3.0

Thanks in advance

**Comment by chaiwa-berian** [comment]
@StephaneMartignier After carefully looking at the code, I noticed that the code on [line number 66 of the index.js](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L66) could potentially be the cause of the error. Here is the reason:

If you run `run-rs -v 4.0.2 ---shell` **without specifying the value** for `--dbpath`, the code will return `./data` as the `dbPath` to be used on line number [69 ](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L69) and line number [80](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L80)!

This will fail on **windows** because `dbPath` will always be **./data** whenever the `--dbpath` option is not passed, and so the code on lines **[69](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L69)** and **[80](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L80)** will always resolve to the path below which is **not a correct syntax for a windows path** that `md` command expects:
`md .\./data` !

To resolve this potential bug for **windows users**, the code on line [66 ](https://github.com/vkarpov15/run-rs/blob/33efa7474781035578003bec6c1eafaead351386/index.js#L66)can be refactored to something like below:
```
const dbPath;
 if(typeof commander.dbpath === 'string'){
   dbPath = `${commander.dbpath}` ;
}
else {
   dbPath = isWin ? 'data' : './data';
}
```
**Note:** _I have not tested this code but I believe it should be able to fix the issue!_ Let me know what you think about this! :)

**Comment by chaiwa-berian** [comment]
I have submitted a pull request for this issue.  See Pull request #14

**Comment by StephaneMartignier** [comment]
I also think that your pr will solve the issue.

Also, in the documentation for the --dbpath option it's written "dbPath" in the example command. It's should be "dbpath"

**Comment by vkarpov15** [comment]
Merged @chaiwa-berian 's PR and released 0.3.2 :+1:
